src/Controls/src/Core/HandlerImpl/Toolbar/Toolbar.Windows.cs

```diff
@@ -46,7 +46,7 @@ internal void UpdateMenu()
 				{
 					button.SetBinding(AppBarButton.IconProperty, "IconImageSource", _imageSourceIconElementConverter);
 				}
-				else
+				else if (!item.IconImageSource.IsNullOrEmpty())
```

Comment:
should we add another else here that nulls out the "content" on the button?

Reply:
Not sure if I got that